tests/python/pants_test/util/test_meta.py

```diff
@@ -31,9 +31,183 @@ def method(self):
       AbstractMethod()
 
 
-class SingletonTest(unittest.TestCase):
+class SingletonTest(TestBase):
```

Comment:
For the record, it's totally kosher to extend `unittest.TestCase` (and a good signal to a reader) if you don't have need of a pants workspace. Not a big deal either way (and not worth another CI run).

Reply:
Noted! I can totally understand that distinction.